client/my-sites/scan/wpcom-scan-upsell.tsx

```diff
@@ -61,7 +61,6 @@ const ScanVPActiveBody: FunctionComponent = () => {
 					className="scan__wpcom-cta"
 					href="https://dashboard.vaultpress.com"
 					onClick={ onUpgradeClick }
-					selfTarget={ true }
```

Comment:
This prop does not exist on `Button`. It was a refactor typo when this was converted from `PromoCardCta`.